src/App.js

```diff
@@ -1,17 +1,50 @@
 import React, { useState } from "react"
 import './App.css';
 import Button from './component/Button';
+import ToDoForm from "./component/TodoForm";
 
+function App() {
 
 
+  const [clickedButton, setClickedButton] = useState(false)
+  const[todos, setTodo] = useState([])
+ 
 
+  const handleClick =  (buttonId) => {
```

Comment:
The function name is too generic.
rename something like 'layoutToggleButtonClick'

Reply:
ok